utils/freifunk-berlin-network-defaults/uci-defaults/freifunk-berlin-default-txpower

```diff
@@ -0,0 +1,33 @@
+#!/bin/ash
+#
+# On Ubiquiti it has been seen that the default TX-power on OpenWRT
+# is at setting of Hardware, which is above regulatory limits
+# see https://github.com/freifunk-berlin/firmware/issues/381
+#
+
+. /lib/functions/guard.sh
+guard "2GHz_txpower"
```

Comment:
the "iwinfo"-method adjusts against the defined offsets in http://git.openwrt.org/?p=project/iwinfo.git;a=blob;f=hardware.txt;h=81b31cfcbca1d00cfabbdb6768ab9e0f97a8cc89;hb=d22c64c2eb6db3db6aaac5ae270acd63ed088a20.
So this works on more hardware than just the NanoStations, but infact atm only works for Ubiquiti. But this can change by the next releases of iwinfo.
So "2GHz_txpower" is currently maybe a bit wide, but "ubnt_nsm_24ghz_txpower" is a bit to specific.
If we want to narrow down we can do something like "ubnt_24ghz_txpower", which reflects the current hardware-list